app/src/fenixProduction/res/values/static_strings.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<resources>
+    <!-- Name of the application -->
+    <string name="app_name" translatable="false">Firefox Nightly</string>
```

Comment:
It looks like we'll be using the brand name "Firefox Nightly"!

I can't see the assets though, so I want to confirm that we're going to be shipping with the Firefox Nightly assets/branding (which is on fennecNightly flavor) and NOT the Firefox Preview assets/branding (which has been the fenixProduction flavor)

cc @pocmo @JohanLorenzo

Reply:
@liuche It looks like we copied the fenixNightly assets over, but we may actually want the fennecNightly assets (outline logo vs. full logo)?

Reply:
Discussed on Slack with @liuche, yesterday. Fixed in https://github.com/mozilla-mobile/fenix/pull/12173.